game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -453,7 +450,38 @@ function Duels:AllPlayers (state, cb)
   end
 end
 
+function Duels:SafeTeleportAll(owner, location, maxDistance)
+  self:SafeTeleport(owner, location, maxDistance)
+  local children = FindUnitsInRadius(owner:GetTeam(),
+                                     owner:GetAbsOrigin(),
+                                     nil,
+                                     FIND_UNITS_EVERYWHERE,
+                                     DOTA_UNIT_TARGET_TEAM_FRIENDLY,
+                                     DOTA_UNIT_TARGET_BASIC,
+                                     DOTA_UNIT_TARGET_FLAG_PLAYER_CONTROLLED,
+                                     FIND_ANY_ORDER,
+                                     false)
+  for _,child in pairs(children) do
+    if child:HasMovementCapability() then
+      self:SafeTeleport(child, location, maxDistance)
```

Comment:
Don't you need to confirm that unit belongs to that hero, or you'll end up with all of the team's units on top of whoever was teleported in last?